clang/include/clang/Driver/Options.td

```diff
@@ -2085,6 +2085,8 @@ defm prebuilt_implicit_modules : BoolFOption<"prebuilt-implicit-modules",
   HeaderSearchOpts<"EnablePrebuiltImplicitModules">, DefaultFalse,
   PosFlag<SetTrue, [], "Look up implicit modules in the prebuilt module path">,
   NegFlag<SetFalse>, BothFlags<[NoXarchOption, CC1Option]>>;
+def fmodule_related_to_pch : Flag<["-"], "fmodule-related-to-pch">,
+  Group<f_Group>, Flags<[CC1Option]>, HelpText<"Mark module as related to a PCH">;
```

Comment:
Wondering if `-fmodule-context-hash-uses-pch` would be more clear?

Reply:
Hmm, I'm not sure. This doesn't only change the module context hash: the main purpose is to modify `LangOptions`. The different context hash this will result in is only a secondary effect we don't particularly care about during explicit build anyways. The `uses-pch` spelling might be confusing, since the module that will be explicitly built with this flag isn't using a PCH.

Reply:
I can clarify what "related to a PCH" means in the `HelpText` though.

Reply:
Maybe `-fmodules-context-hash-pch-build-arena`?